### ISSUE TITLE
Whats the icelandic phrase for compound interest calculator? Aka (also known as) it would be... what?

### ISSUE THREAD

**Issue by judyfong** [afk0901/Compound_Interest_Calculator#4]
Its taught in the rekstrarsbókhald course by runar steinn. A competitor is meniga.

**Comment by afk0901** [comment]
I have no idea, but the Icelandic translation of the word would be: "Reiknivél fyrir samsetta vexti". I don't use Meniga so I don't know if there is a compound interest calculator there or not so I don't know what it's called in Icelandic there.

**Comment by judyfong** [comment]
I think there isnt a difference in icelandic between simple and compounding just called vextir reiknavel https://lanareiknivel.is

**Comment by afk0901** [comment]
Yeah, I mean "Vaxtareiknivél" also makes sense. It's just more generic.
